maintenance/2023-10-29.md

```diff
@@ -0,0 +1,13 @@
+## API maintenance
+
+On Sun, 2023-10-29, between 05:00 and 10:00 UTC Database service for [Foundries.io API](https://api.foundries.io/ota/) will be reconfigured. During the reconfiguration, some calls to the API might fail.
+
+Affected calls:
+ - device list
+ - waves list
+ - target tests
+ - device tests
+
+As a consequence, web application [app.foundries.io](https://app.foundries.io/) might diplay errors.
+
+Expected downtime - less than 5 minutes in a row.
```

Comment:
"in a row" seems complicated.  Does this mean that they might see several bits of downtime each not to exceed 5 minutes?

Reply:
If that is the case then it should be "not to exceed 5 minutes at a time"

Reply:
> "in a row" seems complicated. Does this mean that they might see several bits of downtime each not to exceed 5 minutes?

its correct